pandas/core/strings.py

```diff
@@ -197,8 +196,8 @@ def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
 
 def str_startswith(arr, pat, na=np.nan):
     """
-    Return boolean array indicating whether each string starts with passed
-    pattern
+    Return boolean Series indicating whether each string starts with passed pattern.
+    Equivalent to ``str.startswith(pat)``.
```

Comment:
Ah ok, I see, you mean the standard library's method on `str` ..! 
I think the following should also work ànd create a link to the python docs: ```:meth:`str.startswith` ``` 
And maybe use "equivalent to the standard str.startswith" ? (although I am not really satisfied with that wording as well)